.editorconfig

```
@@ -0,0 +1,9 @@
+[*]
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+indent_size = 4
+
+[*.md]
+trim_trailing_whitespace = false
```

We don't need markdown to be special, but we do need to indent Makefiles with tabs. There's also one test file that is indented with a tab.

```suggestion
[{Makefile,Makefile.*,tests/syntax_error/tab.jou}]
indent_style = tab
```